### ISSUE TITLE
[BUG] Tooltips on subgroup images render as [object Object]

### ISSUE THREAD

**Issue by sferguson12** [Larkinabout/fvtt-token-action-hud-core#291]
**Describe the bug**

When hovering over the images displayed next to Strike subgroups, the tooltip renders incorrectly as `[object Object]` instead of the actual tooltip.

![image](https://github.com/user-attachments/assets/caab2a97-a147-4f30-b925-d66f987978d1)

Since the tooltip is being converted to an object in `getTooltip()` in `tooltip-handler.mjs`, this appears to be an issue in the `list-subgroup.hbs` template in how the tooltip is being set.

```diff
{{~#if (ne selected false)~}}
<div class="tah-subgroup tah-list-subgroup" data-nest-id="{{nestId}}" data-part="subgroup" data-has-actions="{{~#if (tokenActionHudCoreTrue hasActions)~}}true{{~else~}}false{{~/if~}}" data-has-image="{{~#if (tokenActionHudCoreTrue settings.image)~}}true{{~else~}}false{{~/if~}}" data-level="{{level}}" data-type="{{type}}" data-show-title="{{settings.showTitle}}">
-    {{~#if settings.image~}}<img class="tah-list-image{{~#if (tokenActionHudCoreTrue settings.collapse)}} tah-hidden{{~/if}}" src="{{settings.image}}" {{~#if tooltip~}} data-tooltip="{{tooltip}}" data-tooltip-class="tah-tooltip"{{~/if~}}>{{~/if~}}
+    {{~#if settings.image~}}<img class="tah-list-image{{~#if (tokenActionHudCoreTrue settings.collapse)}} tah-hidden{{~/if}}" src="{{settings.image}}" {{~#if tooltip~}} data-tooltip="{{tooltip.content}}" data-tooltip-class="tah-tooltip"{{~/if~}}>{{~/if~}}
    <div class="tah-list-subgroup{{~#if class}} {{class}}{{~/if~}}" data-part="listSubgroup">
        <div class="tah-list-subgroup-title{{~#if subtitleClass}} {{subtitleClass}}{{~/if~}}" data-action="clickSubgroup" data-part="listSubgroupTitle">
```

The current state results in this being added to the DOM:

```html
<img class="tah-list-image" src="systems/pf2e/icons/unarmed-attacks/jaws.webp" data-tooltip="[object Object]" data-tooltip-class="tah-tooltip"><div class="tah-list-subgroup" data-part="listSubgroup">
```

Which renders as this on hover:

```html
<aside id="tooltip" role="tooltip" data-theme="crb" style="top: 348.812px; left: 168.5px;" class="tah-tooltip text-center">[object Object]</aside>
```

This issue also appears to exist in `tab-subgroup.hbs` and `group.hbs` if an image is present.

**Steps to reproduce**
1. Load a scene with an actor that has one or more Strike actions with icons
2. Select the actor
3. Wait for the HUD to load
4. Mouse over `Strikes`
5. Mouse over the image for a specific Strike

Expected: The tooltip for that Strike should render
Actual: A tooltip containing "[object Object]" renders

**Versions:**
- Token Action HUD Core: 2.0.6
- Token Action HUD System: token-action-hud-pf2e 2.0.2
- System: pf2e 6.6.2
- Foundry VTT: 12.331
- Browser: Brave 1.71.123 (also reproducible in Chrome and Edge)

**Comment by Larkinabout** [comment]
Fixed in 2.0.7. Thanks for looking into it.

**Comment by sferguson12** [comment]
Not sure if this slipped by in the report above, but I noticed the same issue in `tab-subgroup.hbs` and `group.hbs` and it looks like only `list-subgroup.hbs` was updated in the [commit](https://github.com/Larkinabout/fvtt-token-action-hud-core/commit/7055fd15357e4bdeb832cb57111ec94514eb265a).
